website/docs/d/cos_bucket.html.markdown

```diff
@@ -1,11 +1,12 @@
 ---
-subcategory: "Object Storage"
+subcategory: "Object Storage" , "COS Satellite"
 layout: "ibm"
-page_title: "IBM : Cloud Object Storage Bucket"
+page_title: "IBM : Cloud Object Storage Bucket" , "COS Satellite Bucket"
 description: |-
-  Get information about IBM Cloud Object Storage bucket.
+  Get information about COS Satellite bucket. , "Get information about COS Satellite bucket."
 ---
 
+
 # ibm_cos_bucket
 
 Creates an IBM Cloud Object Storage bucket. It also allows object storage buckets to be updated and deleted. The ibmcloud_api_key used by Terraform must have been granted sufficient IAM rights to create and modify IBM Cloud Object Storage buckets and have access to the Resource Group the Cloud Object Storage bucket will be associated with. See https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-iam for more details on setting IAM and Access Group rights to manage COS buckets.
```

Comment:
Edit this description to say this resource works for IBM cloud COS and Satellite based COS